apps/mobile-wallet/src/screens/BackupMnemonic/VerifyMnemonicScreen.tsx

```diff
@@ -74,17 +76,24 @@ const VerifyMnemonicScreen = ({ navigation, ...props }: VerifyMnemonicScreenProp
 
   const confirmBackup = useCallback(async () => {
     if (!isMnemonicBackedUp) {
-      await updateStoredWalletMetadata({ isMnemonicBackedUp: true })
-      dispatch(mnemonicBackedUp())
+      try {
+        await updateStoredWalletMetadata({ isMnemonicBackedUp: true })
+        dispatch(mnemonicBackedUp())
 
-      sendAnalytics('Backed-up mnemonic')
+        sendAnalytics({ event: 'Backed-up mnemonic' })
+      } catch (error) {
+        const message = 'Could not confirm backup'
+
+        showExceptionToast(error, message)
+        sendAnalytics({ type: 'error', error, message })
```

Comment:
Double checking: no risk of leaking the mnemonic here?

Reply:
No risk